draft-quic-atsss-reqs.mkd

```diff
@@ -706,10 +708,17 @@ Volume-based Quota
 
 Wireless LAN (WLAN)
 
-# Conclusion {#conclusion}
+# QUIC Version 1 Gap Analysis for ATSSS Version 2 {#gap-analysis}
+
+These are additional QUIC protocol capabilities that would be useful for ATSSS Phase 2, as described in {{sec-ver2}}, along with our understanding of their maturity level.
+
+1. ATSSS Version 2 will need transport services for Ethernet frames and IP packets, with the intention of supporting TCP, UDP, and UDP-encapsulated transport protocols such as QUIC and RTP. QUIC Version 1 provides transport services for HTTP/3 but does not yet provide support for tunnelling Ethernet or IP directly over QUIC, as envisioned in ATSSS Version 2. QUIC Version 1. The QUIC working group is not chartered to provide this capability, although proposals exist.
+
+1. QUIC Version 1 does not yet provide a multipath transport capability similar to Multipath TCP, as used in ATSSS Phase 1. The QUIC working group is chartered to produce a multipath extension document by December 2021. Proposals exist, although the working group has not yet adopted a draft as the basis for its work. 
```

Comment:
Is that 1. part of an enumeration ?

Reply:
No, just part of Spencer doing a PR too late at night. Deleted :-)